test/com/facebook/buck/java/intellij/ProjectIntegrationTest.java

```diff
@@ -443,19 +443,32 @@ public void testBuckProjectWithMultipleLibrariesInOneBuildFile() throws IOExcept
   }
 
   @Test
-  public void testNonexistentTarget() throws IOException {
+  public void testProjectWithColon() throws IOException {
     ProjectWorkspace workspace = TestDataHelper.createProjectWorkspaceForScenario(
         this,
         "project1",
         temporaryFolder);
     workspace.setUp();
 
-    expectedException.expect(HumanReadableException.class);
```

Comment:
The PMD error is fixed. Would you please let us know the official build command for building buck? I typically do: `buck test`, which does not run static analysis I think.